README.md

```diff
@@ -1,25 +1,26 @@
-# Installer for qbittorrent (Dark version)
+# Installer for qbittorrent (Dark Theme)
 
 ## Steps
 
-### Archlinux
-1. Use yaourt -S qbittorrent-dark-git
+### Arch Linux
+1. Use `yaourt -S qbittorrent-dark-git`
 
-### Build steps for other distros Linux, MacOS
-1. If you are using this OSes, you know what to do
+### OS X or other Linux Distro
```

Comment:
OS X is a former macOS name, so `macOS` is the most correct case

Reply:
It explains nothing, because since Sierra release OS X changed it's name to macOS.
Trust me, I'm using this OS for 5 years :)

Reply:
To provide you some proofs here is part of my MacBook's "System Report" page
![image](https://user-images.githubusercontent.com/3203317/51503152-860c0c00-1dea-11e9-929b-035725db8690.png)